aepsych/server/server.py

```diff
@@ -421,7 +450,9 @@ def handle_setup(self, request):
             "config_str" in request["message"].keys()
             or "config_dict" in request["message"].keys()
         ):
-            _ = self.configure(**request["message"])
+            # Generate a config object since configure will just accept one directly.
```

Comment:
The old way should still work, right? So is this change necessary?

Reply:
Good point! I'll revert it back since it's an unnecessary step. Good catch 👍